baybe/kernels/kernels.py

```diff
@@ -0,0 +1,18 @@
+"""Collection of usable kernels."""
+
+from attrs import define, field
+from attrs.validators import in_
+
+from baybe.kernels.base import Kernel
+
+
+@define
+class MaternKernel(Kernel):
+    """A Matern kernel using a smoothness parameter nu."""
```

Comment:
Let's not put parameter names into docstrings (refactorability)

Reply:
Changed.